packages/main/src/errors.ts

```diff
@@ -31,3 +40,30 @@ export class GoogleGenerativeAIResponseError<
     super(message);
   }
 }
+
+/**
+ * Error class covering http errors when calling the server. Includes http
```

Comment:
We seem to capitalize `http` in our Firebase JS SDK's docs:

[See here](https://firebase.google.com/docs/reference/node/firebase.functions.HttpsError#code).